chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -1,36 +1,43 @@
+TODO: https://github.com/filecoin-project/lotus/pull/12450#discussion_r1772125029
+move this to lotus-docs
+
 # ChainIndexer Documentation for RPC Providers
   
 ## Introduction
 
-This document is for RPC providers and node operators who serve RPC requests walking through the configuration changes, migration flow and operations/maintenance work needed to enable, backfill and maintain the [`ChainIndexer`](#chainindexer-indexing-system).
+This document is for externally-available and/or high-performance RPC providers.  It walks through the configuration changes, migration flow and operations/maintenance work needed to enable, backfill and maintain the [`ChainIndexer`](#chainindexer-indexing-system).  The justification for and benefits of the `ChainIndexer` are documented [here](https://github.com/filecoin-project/lotus/issues/12453). 
 
-**Note: If you are a Storage Provider or node operator who does not serve RPC requests (i.e, if `Fevm.EnableEthRPC = false`), you can skip this document as the `ChainIndexer` is already disabled by default**. 
+**Note: If you are a Storage Provider or node operator who does not serve public RPC requests (i.e, if `Fevm.EnableEthRPC = false`), you can skip this document as the `ChainIndexer` is already disabled by default**. 
 
 ## ChainIndexer Config
 ### Enablement
 
 The following must be enabled on an Lotus node before starting as they are disabled by default:
 
 ```toml
-[ChainIndexer]
-# Enable the ChainIndexer. 
-  EnableIndexer = true 
-
 [Fevm]
 # Enable the ETH RPC APIs.
```

Comment:
```suggestion
# Enable the ETH RPC APIs.
# This is not required for ChainIndexer support, but ChainIndexer is required if you enable this.
```

the nuance here is that ChainIndexer helps Eth APIs but you can use ChainIndexer to speed up message lookup and the Get/SubscribeActorEventsRaw APIs.